SGM/src/sgmstereo/dsimage.h

```diff
@@ -176,13 +186,17 @@ class DSI
 
 	~DSI()
 	{
-		free();
+		freeMem();
 	}
 
-	void free()
+	void freeMem()
```

Comment:
Nitpick: would you mind fixing the indentation so that it's consistent in this function while we're making change?